File: app/src/main/java/com/example/dishora/defaultUI/profileTab/startSellingOption/steps/dto/BusinessDto.java
package com.example.dishora.defaultUI.profileTab.startSellingOption.steps.dto;

public class BusinessDto {
    private String businessName;
    private String description;
    private String type;
    private String location;
    private String businessDuration;
    private String birRegNo;
    private String businessPermitNo;
    private String validIdType;
    private String validIdNo;
    private double latitude;
    private double longitude;

    // ===== Getters/Setters =====

    public String getBusinessName() { return businessName; }
    public void setBusinessName(String businessName) { this.businessName = businessName; }

    public String getDescription() { return description; }
    public void setDescription(String description) { this.description = description; }

    public String getType() { return type; }
    public void setType(String type) { this.type = type; }

    public String getLocation() { return location; }
    public void setLocation(String location) { this.location = location; }

    public double getLatitude() { return latitude; }
    public void setLatitude(double latitude) { this.latitude = latitude; }

    public double getLongitude() { return longitude; }
    public void setLongitude(double longitude) { this.longitude = longitude; }

    public String getBusinessDuration() { return businessDuration; }
    public void setBusinessDuration(String businessDuration) { this.businessDuration = businessDuration; }

    public String getBirRegNo() { return birRegNo; }
    public void setBirRegNo(String birRegNo) { this.birRegNo = birRegNo; }

    public String getBusinessPermitNo() { return businessPermitNo; }
    public void setBusinessPermitNo(String businessPermitNo) { this.businessPermitNo = businessPermitNo; }

    public String getValidIdType() { return validIdType; }
    public void setValidIdType(String validIdType) { this.validIdType = validIdType; }

    public String getValidIdNo() { return validIdNo; }
    public void setValidIdNo(String validIdNo) { this.validIdNo = validIdNo; }
}

File: DTO/BusinessDto.cs
namespace Dishora.DTO
{
    public class BusinessDto
    {
        public string businessName { get; set; }
        public string description { get; set; }
        public string type { get; set; }
        public string location { get; set; }
        public double latitude { get; set; }
        public double longitude { get; set; }

        public string? businessImage { get; set; }
        public string? businessDuration { get; set; }

        public string? birRegNo { get; set; }
        public string? birRegFile { get; set; }
        public string? businessPermitNo { get; set; }
        public string? businessPermitFile { get; set; }
        public string? validIdType { get; set; }
        public string? validIdNo { get; set; }
        public string? validIdFile { get; set; }
        public string? mayorPermitFile { get; set; }

        public List<OpeningHourDto> OpeningHours { get; set; }
    }
}
